examples/dynamics/throwing_while_standing_on_the_ice.py

```diff
@@ -0,0 +1,61 @@
+from sympy import solve
+from symplyphysics import print_expression, units, Symbol, dimensionless
+from symplyphysics.laws.dynamics import friction_force_from_normal_force as friction_force
+from symplyphysics.laws.dynamics import mechanical_work_from_force_and_move as work_friction
+from symplyphysics.laws.dynamics import kinetic_energy_from_mass_and_velocity as kinetic_energy
+from symplyphysics.definitions import momentum_is_mass_times_velocity as impuls
+from symplyphysics.laws.conservation import momentum_after_collision_equals_to_momentum_before as impuls_conservation
+from symplyphysics.laws.conservation import mechanical_energy_after_equals_to_mechanical_energy_before as energy_conservation
+
+# Example from https://uchitel.pro/%D0%B7%D0%B0%D0%B4%D0%B0%D1%87%D0%B8-%D0%BD%D0%B0-%D0%B7%D0%B0%D0%BA%D0%BE%D0%BD-%D1%81%D0%BE%D1%85%D1%80%D0%B0%D0%BD%D0%B5%D0%BD%D0%B8%D1%8F-%D0%B8%D0%BC%D0%BF%D1%83%D0%BB%D1%8C%D1%81%D0%B0/
+# A skater with a mass of M = 70 kg, standing on the ice,
+# throws a puck with a mass of m = 0.3 kg horizontally at a speed of v = 40 m/s.
+# How far s will the skater roll back if the coefficient of friction of the skates
+# on the ice is μ = 0.02?
+
+mass_of_skater = Symbol("mass_of_skater", units.mass)
+mass_of_puck = Symbol("mass_of_puck", units.mass)
+friction_factor = Symbol("friction_factor", dimensionless)
+velocity_of_puck = Symbol("velocity_of_puck", units.velocity)
+distance = Symbol("distance", units.length)
+
+gravity_acceleration = Symbol("gravity_acceleration", units.acceleration)
+
+velocity_of_skater = Symbol("velocity_of_scater", units.velocity)
+
+impuls_of_skater = impuls.definition.subs({
+    impuls.mass: mass_of_skater,
+    impuls.velocity: velocity_of_skater
+}).rhs
+impuls_of_puck = impuls.definition.subs({
+    impuls.mass: mass_of_puck,
+    impuls.velocity: velocity_of_puck
+}).rhs
+
+impuls_conservation_law = impuls_conservation.law.subs({
+    impuls_conservation.momentum(impuls_conservation.time_before): 0,
+    impuls_conservation.momentum(impuls_conservation.time_after): impuls_of_skater - impuls_of_puck
+})
+velocity_of_skater_law = solve(impuls_conservation_law, velocity_of_skater, dict=True)[0][velocity_of_skater]
+
+friction_force_law = friction_force.law.subs({
+    friction_force.friction_factor: friction_factor,
+    friction_force.normal_reaction: mass_of_skater * gravity_acceleration
```

Comment:
To be really precise, one could use Newton's second law: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/dynamics/force_reaction_from_force_action.py
But I do not think it is all that important here.

Reply:
If we find the distance traveled by the skater after the throw through the equation of motion with constant acceleration, Newton's 2nd law in projection onto the horizontal would definitely be useful, because we would need acceleration. And then we would definitely have to use this module that you specified. In the proposed solution, the acceleration (more precisely, the deceleration)  of the skater is not necessary for us, I suppose.

Reply:
Sorry I meant 3rd law. As friction works on reaction force, we should not insert action force in the friction formula.

Reply:
You mean this?
```python
from symplyphyics.laws import force_reaction_from_force_action as third_newton_law
from symplyphysics.laws.dynamics import friction_force_from_normal_force as friction_force
 
# \vector{N} = - m \vector{g}
reaction_force_value = third_newton_law.law.subs({
    third_newton_law.force_action: -mass_of_skater * gravity_acceleration
}).rhs

friction_force_value = friction_force.law.subs({
    friction_force.friction_factor: friction_factor,
    friction_force.normal_reaction: reaction_force
}).rhs
```

Reply:
Yes, but I don't think it is really necessary.

Reply:
I added this because it may not be entirely obvious to the reader of the code in this example why the reaction force is $N =m_{skater} * g$ in the friction force formula. And by adding this, there will be a hint of a connection with Newton's laws.